test/gtest/uct/ib/test_ib.cc

```diff
@@ -333,6 +333,59 @@ UCS_TEST_P(test_uct_ib, address_pack) {
 UCT_INSTANTIATE_IB_TEST_CASE(test_uct_ib);
 
 
+class test_uct_ib_utils : public ucs::test {
+};
+
+UCS_TEST_F(test_uct_ib_utils, sec_to_qp_time) {
+    uint8_t qp_val;
+
+    qp_val = uct_ib_to_qp_fabric_time(0);
+    EXPECT_EQ(1, qp_val);
+
+    for (uint8_t i = 1; i < 32; i++) {
+        qp_val = uct_ib_to_qp_fabric_time(4.096 * pow(2, i) / UCS_USEC_PER_SEC);
+        EXPECT_EQ(i % 31, qp_val);
+    }
+
+    qp_val = uct_ib_to_qp_fabric_time(10000);
+    EXPECT_EQ(0, qp_val);
+}
+
+UCS_TEST_F(test_uct_ib_utils, sec_to_rnr_time) {
+    double avg;
+    uint8_t rnr_val;
+
+    rnr_val = uct_ib_to_rnr_fabric_time(0);
+    EXPECT_EQ(1, rnr_val);
+
+    rnr_val = uct_ib_to_rnr_fabric_time(uct_ib_qp_rnr_time_ms[0] / UCS_MSEC_PER_SEC);
```

Comment:
what do you want to check here? You checked that 0 and minimum values produce the same result - 1.
maybe better check average between table[0] and table[1]?